tools/devops/automation/scripts/update-locproject.ps1

```diff
@@ -15,10 +15,14 @@ $projectObject = Get-Content $LocProjectPath | ConvertFrom-Json
 $jsonFiles | ForEach-Object {
     $sourceFile = $_.FullName
     $outputPath = "$($_.DirectoryName + "\")"
+    $fullNameString = Convert-Path -Path $_.FullName
+    $maciosPath = $fullNameString -split "xamarin-macios", 2
+    $lclFile = "$($LocalizeDirectory + $maciosPath[1] + ".lcl")"
```

Comment:
This is going to be a problem if the split did not return with 2 items in it. To avoid  surprises, test the length, if it is not the correct, exit 1 stating the reason. Otherwise we might get a $null. Pwsh does not throw exceptions when you index wrongly an array, for example:
```pwsh
$foo = @("one", "two")
$result = $foo[3]
if ($null -eq $result) {
    Write-Host "You've got a null value!"
}
```
Because you are getting null, you get the wrong path.